pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestDataAndQuerySet.java

```diff
@@ -52,14 +52,18 @@
  * Shared set of common tests for cluster integration tests.
  * <p>To enable the test, override it and add @Test annotation.
  */
-public abstract class BaseClusterIntegrationTestSet extends BaseClusterIntegrationTest {
-  private static final Logger LOGGER = LoggerFactory.getLogger(BaseClusterIntegrationTestSet.class);
+public abstract class ClusterIntegrationTestDataAndQuerySet extends ClusterIntegrationTestDataSet {
```

Comment:
Should we make it similar to `ClusterIntegrationTestDataSet` and move the default settings into a separate class?